torch/CMakeLists.txt

```diff
@@ -242,6 +242,12 @@ if (USE_NCCL)
     endif()
 endif()
 
+if (BUILD_PROTOTYPES)
+    list(APPEND TORCH_PYTHON_COMPILE_DEFINITIONS BUILD_PROTOTYPES)
```

Comment:
What does this line do?

Reply:
Adds -DBUILD_PROTOTYPES so it can be used during compile-time

Reply:
That's amazing, I didn't know that's how that worked